### PR TITLE
Update /chinese/fcitx to 4.2.9.9 and update /chinese/libime to 1.1.9

### PR DESCRIPTION
**Pull request by tempcracks**
- [x] tested on 14.1-RELEASE

### CODE REVIEW

File: chinese/fcitx/Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 PORTNAME=	fcitx
-PORTVERSION=	4.2.9.8
-PORTREVISION=	2
```
Comment (lwhsu) [inline chinese/fcitx/Makefile:3]: just remove `PORTREVISION` when `PORTVERSION` gets updated.

File: chinese/libime/Makefile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 PORTNAME=	libime
-DISTVERSION=	1.1.8
-PORTREVISION=	1
```
Comment (lwhsu) [inline chinese/libime/Makefile:3]: just remove `PORTREVISION` when `PORTVERSION` gets updated.

**Comment by lwhsu** [comment]
@lichray

File: chinese/fcitx/Makefile [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+PORTVERSION=	4.2.9.9
+PORTREVISION=	3
```
Comment (lwhsu) [inline chinese/fcitx/Makefile:3]: just remove `PORTREVISION` when `PORTVERSION` gets updated.

File: chinese/libime/Makefile [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
+DISTVERSION=	1.1.9
+PORTREVISION=	2
```
Comment (lwhsu) [inline chinese/libime/Makefile:3]: just remove `PORTREVISION` when `PORTVERSION` gets updated.

**Review by lwhsu — CHANGES_REQUESTED** [review]

File: chinese/fcitx/Makefile [hunk 5]
```diff
@@ -1,6 +1,6 @@
 PORTNAME=	fcitx
-PORTVERSION=	4.2.9.8
-PORTREVISION=	2
+PORTVERSION=	4.2.9.9
+PORTREVISION=
```
Comment (lichray) [inline chinese/fcitx/Makefile:3]: Just delete the lines.

Comment (tempcracks) [reply chinese/fcitx/Makefile:3]: ok, sorry

**Review by lichray — APPROVED** [review]

**Comment by tempcracks** [comment]
@lwhsu can you approve?

**Review by lwhsu — APPROVED** [review]

**Comment by lwhsu** [comment]
Please help to split the updating of these 2 ports in 2 commits so it's easier for me to update metadata and push. Thanks.

**Comment by tempcracks** [comment]
@lwhsu  everything's all right

**Comment by tempcracks** [comment]
@lwhsu will you merge? Thank you

**Comment by lwhsu** [comment]
> @lwhsu everything's all right?

Yes, thanks.

> @lwhsu will you merge? Thank you

Yes. I'll do some processing and merge. Please give me some more time.

**Pull request closed without merging**